packages/ckeditor5-table/src/tableselection/croptable.js

```diff
@@ -8,6 +8,36 @@
  */
 
 import { findAncestor } from '../commands/utils';
+import TableWalker from '../tablewalker';
+
+/**
+ * Returns a cropped table according to given dimensions.
+ *
+ * This function is to be used with the table selection.
+ *
+ *		const croppedTable = cropTable( table, 1, 1, 3, 3, tableUtils, writer );
+ *
+ * @param {Number} sourceTable
+ * @param {Number} startRow
+ * @param {Number} startColumn
+ * @param {Number} endRow
+ * @param {Number} endColumn
+ * @param {module:table/tableutils~TableUtils} tableUtils
+ * @param {module:engine/model/writer~Writer} writer
+ * @returns {module:engine/model/element~Element}
+ */
+export function cropTableToDimensions( sourceTable, startRow, startColumn, endRow, endColumn, tableUtils, writer ) {
+	const tableCopy = makeTableCopy( sourceTable, startRow, startColumn, endRow, endColumn, writer, tableUtils );
```

Comment:
Both functions have very similar arguments list except the last two.

Reply:
I'll fix that.

Reply:
I've refactored this completely as I saw some duplications in loops - the code is now more compact.